src/pybind/mgr/cephadm/module.py

```diff
@@ -1514,7 +1514,12 @@ def _run_cephadm(self, host, entity, command, args,
             addr = self.inventory[host].get('addr', host)
 
         try:
-            conn, connr = self._get_connection(addr)
+            try:
+                conn, connr = self._get_connection(addr)
+            except IOError as e:
+                if error_ok:
+                    return [], [str("Can't communicate with remote host, possibly because python3 is not installed there")], 1
```

Comment:
```suggestion
                    logger.exception('failed to establish ssh connection')
                    return [], [str(f"Can't communicate with remote host {host}, possibly because python3 is not installed there")], 1
```

Reply:
:+1: